django-server/server/api/views.py

```diff
@@ -112,6 +112,131 @@ def get_queryset(self):
         return FixedMenuOptionReview.objects.all().filter(option_id=self.kwargs['option_pk']).order_by('id')
 
 
+class TimeStampedFilterSet(filters.FilterSet):
+    timestamp_start = filters.DateTimeFilter(required=True)
+    count = filters.NumberFilter(required=True)
+
+
+class StatsView(generics.ListAPIView):
+    serializer_class = OccupancyStatsSerializer
+    filter_backends = [filters.DjangoFilterBackend]
+    filterset_class = TimeStampedFilterSet
+
+    def get_queryset(self):
+        if getattr(self, 'swagger_fake_view', False):
+            # queryset just for schema generation metadata
+            return []
+        id = self.kwargs.get('cafeteria_pk')
+        count = self.request.query_params.get('count')
+        timestamp_start = self.request.query_params.get('timestamp_start')
+        if id is None or count is None or timestamp_start is None:
+            raise ValueError('required params not specified')
+
+        timestamp_start = parse(timestamp_start)
```

Comment:
Please either use two variables or move it to line 132, right after original definition.

Reply:
fixed